models/mux_protocol/optimism/mux_protocol_optimism_perpetual_trades.sql

```diff
@@ -0,0 +1,148 @@
+{{ config(
+    
+    alias = 'perpetual_trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
+    post_hook='{{ expose_spells(\'["optimism"]\',
+                                "project",
+                                "mux_protocol",
+                                \'["sofiat"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2023-01-05' %}
+
+WITH 
+
+open_position as (
+SELECT
+    evt_block_time,
+    contract_address,
+    evt_tx_hash as tx_hash,
+    evt_index,
+    trader,
+    'open' as trade_type,
+    CAST(json_extract_scalar(args, '$.collateralId') as double) as underlying_id,
+    assetId as virtual_id,
+    CAST(json_extract_scalar(args, '$.isLong') as varchar) as direction,
+    CAST(json_extract_scalar(args, '$.feeUsd') as double)/1e18 as fee_usd,
+    CAST(json_extract_scalar(args, '$.amount') as double)/1e18 as volume_usd,
+    CAST(json_extract_scalar(args, '$.amount') as double) as volume_raw,
+    CAST(NULL as varbinary) as margin_asset
+FROM
+    {{ source('mux_optimism', 'LiquidityPoolHop1_evt_OpenPosition') }}
+     {% if not is_incremental() %}
+    WHERE evt_block_time >= DATE '{{project_start_date}}'
+    {% endif %}
+    {% if is_incremental() %}
+    WHERE evt_block_time >= date_trunc('day', now() - interval '7' DAY)
+    {% endif %}
```

Comment:
can you update incremental logic across the whole PR to match our new design pattern?
target filter example [here](https://github.com/duneanalytics/spellbook/blob/main/models/_sector/dex/trades/avalanche_c/platforms/openocean_avalanche_c_base_trades.sql#L9)
source filter example [here](https://github.com/duneanalytics/spellbook/blob/main/models/_sector/dex/trades/avalanche_c/platforms/openocean_avalanche_c_base_trades.sql#L30-L32)

be sure the columns passed in for each are correct for each spell